README.md

```diff
@@ -50,7 +50,5 @@ or call `workspace.listen(PORT)`.
 
 ## Test
 
-To run end-to-end tests, you will need a local MySQL instance. Once you have
-that running, you can run [the MySQL setup script](/bin/setup-mysql.js) to
-autoconfigure your database. Finally, use the `npm test` command to run the
-tests.
+**To run end-to-end tests, you will need a local MySQL instance.** Once that is
+running, use the `npm test` command to run the tests.
```

Comment:
This is not correct. You have to run `setup-mysql.js` once to create the test database and user.